test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -216,9 +275,42 @@ describe('CSOT spec prose tests', function () {
        *   - Expect this to fail with a timeout error.
        * 1. Verify that a `find` command was executed against the `keyvault.datakeys` collection as part of the `encrypt` call.
        */
+      it('times out due to timeoutMS', clientEncryptionMetadata, async function () {
+        const datakeyId = await clientEncryption.createDataKey('local');
+        expect(datakeyId).to.be.instanceOf(Binary);
+        expect(datakeyId.sub_type).to.equal(Binary.SUBTYPE_UUID);
+
+        await internalClient
+          .db()
+          .admin()
+          .command({
+            configureFailPoint: 'failCommand',
+            mode: {
+              times: 1
+            },
+            data: {
+              failCommands: ['find'],
+              blockConnection: true,
+              blockTimeMS: 150
+            }
+          } as FailPoint);
+
+        const commandStarted: CommandStartedEvent[] = [];
+        keyVaultClient.on('commandStarted', ev => commandStarted.push(ev));
+
+        const err = await clientEncryption
+          .encrypt('hello', {
+            algorithm: `AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic`,
+            keyId: datakeyId
+          })
+          .catch(e => e);
+
+        expect(err).to.be.instanceOf(MongoOperationTimeoutError);
+        expect(commandStarted[0]).to.containSubset({ commandName: 'find' });
+      });
     });
 
-    context('decrypt', () => {
+    context('decrypt', clientEncryptionMetadata, () => {
```

Comment:
Putting the client metadata here doesn't prevent it from running in environments it shouldn;t, you need it in the `it` call

Reply:
Changed the location!

Reply:
Metadata still here 🙂 

(also, fwiw I _think_ metadata is inherited from a context/describe into any tests that are direct children of the context/describe.  but not any deeper)